x-pack/plugins/ml/public/alerting/jobs_health_rule/register_jobs_health_alerting_rule.ts

```diff
@@ -61,7 +61,14 @@ export function registerJobsHealthAlertingRule(
       {
         defaultMessage: `Anomaly detection jobs health check result:
 \\{\\{context.message\\}\\}
-- Job IDs: \\{\\{context.jobIds\\}\\}
+\\{\\{#context.results\\}\\}
```

Comment:
Some thoughts on the content here:
- Can the logging time be formatted as a date / time?
- Can you add the configured `model_memory_limit` into the message - `model_bytes_memory_limit` from the stats?
- Also I guess it could be useful to display `model_bytes_exceeded` too, or at least stored in the alert doc?
- Should we / can we add a link back to a filtered jobs list, so the user could then explore the stats for this job for example?

![image](https://user-images.githubusercontent.com/7405507/127142140-1ee606d7-a21f-4dd3-90ea-3d185dc5fc9b.png)

Reply:
> - Can the logging time be formatted as a date / time?
AFAIK formatters are not supported in mustache templates. We can format the date in the executor code instead. Do you think we should apply formatting based on the Kibana settings?

> Can you add the configured model_memory_limit into the message - model_bytes_memory_limit from the stats?
> Also I guess it could be useful to display model_bytes_exceeded too, or at least stored in the alert doc?
Added in 94a7d2803c917f044a871fd2644c44c92b50ccd7

Reply:
> AFAIK formatters are not supported in mustache templates. We can format the date in the executor code instead. Do you think we should apply formatting based on the Kibana settings?

Ideally it would be good to use the `dateFormat:tz` Kibana advanced setting to format the logging time, but if that is not possible, formatting using the timezone of the server would be preferable to the timestamp I think, as long as you clearly label the timezone used.

> I reckon we should. Recently Kibana introduces locators instead of URL generators, hopefully, now it's possible to use it server-side to create URLs. It's a fairly big change because it requires moving the ML locator to a common location so I believe it deserves a dedicated PR.

Yes this should be done in a separate PR, with the link to the Jobs list added for all the operational alert messages.